components/sp-solutions/org.wso2.carbon.sp.solutions.ei.analytics/widgets/MessageFlow/mediators/aggregatemediator.svg

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ ~   Copyright (c) 2017 WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ ~
```

Comment:
fix year.

Reply:
Fixed in 53951238f84729f0c325006f8def576a3c702eda